src/tests/validation.test.ts

```diff
@@ -384,12 +384,17 @@ interface TestValidationOptions {
     filterMessages?: string[]
 }
 
-async function testValidations(options: TestValidationOptions) {
-    const { json, diagnostics, filterMessages } = options
+async function getValidations(json: string) {
     const { textDoc, jsonDoc } = toDocument(json);
     const ls = getLanguageService({});
 
-    let res = await ls.doValidation(textDoc, jsonDoc)
+    return await ls.doValidation(textDoc, jsonDoc)
+}
+
+export async function testValidations(options: TestValidationOptions) {
```

Comment:
Why does a test suite need to export a function? Same comment for `testCompletions` exported from `src/tests/completion.test.ts`

Reply:
It's not needed, I forgot to delete it - thank you for spotting it.